### PR TITLE
Features/convert to measurements

### PR DESCRIPTION
**Pull request by rickardenglund**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Build Release
+
+on:
+  push:
+    tags:
+      - "v*"
+
+jobs:
+  test:
```
Comment (j-sv) [inline .github/workflows/release.yml:9]: Should be `release`?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	$(RM) -rf "$@" && $(MKDIR) -p "$@"
+	$(DOCKER) run --rm \
+		--volume "$(shell pwd):/src" \
+		--user "$(shell id -u):$(shell id -g)" \
+		quay.io/goswagger/swagger:v0.25.0 \
+			generate model --spec="/src/$<" --target="/src/$@/.."
+
+rest/swagger.json:
+	$(WGET) "$(API_URL)/docs/swagger/doc.json" -O "$@"
+#	./scripts/patch-skf-uuids.sh "$@"
```
Comment (Ekberg) [inline Makefile:19]: Remove these commented lines?

File: go.mod [hunk 3]
```diff
@@ -1,4 +1,12 @@
-module github.com/SKF/go-hierarchy-client
+module github.com/SKF/go-measurements-client
 
 go 1.15
```
Comment (Ekberg) [inline go.mod:3]: 1.16? :)

File: rest/client.go [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+	"github.com/SKF/go-measurements-client/rest/models"
+
+	rest "github.com/SKF/go-rest-utility/client"
+	"github.com/SKF/go-utility/v2/uuid"
+
+	"github.com/SKF/go-utility/v2/stages"
+)
+
+type TreeFilter struct {
```
Comment (Ekberg) [inline rest/client.go:15]: Remove unused Hierarchy struct

File: rest/client.go [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+	return &client{Client: restClient}
+}
+
+func (c *client) GetNodeDataRecent(ctx context.Context, nodeID uuid.UUID, contentType []string) (models.ModelNodeDataResponse, error) {
+	request := rest.Get("nodes/{nodeID}/node-data/recent{?content_type*}").
+		Assign("nodeID", nodeID.String()).
+		Assign("content_type", contentType).
+		SetHeader("Accept", "application/json").
+		SetHeader("Content-Type", "application/json")
```
Comment (Ekberg) [inline rest/client.go:56]: Content-Type on GET seems strage?

File: rest/example/go.mod [hunk 6]
```diff
@@ -0,0 +1,12 @@
+module github.com/SKF/go-measurements-client/example
+
+go 1.15
```
Comment (Ekberg) [inline rest/example/go.mod:3]: 1.16 if changed above

File: scripts/patch-skf-uuids.sh [hunk 7]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```
Comment (Ekberg) [inline scripts/patch-skf-uuids.sh:1]: Unused script, should be removed.

File: scripts/patch-x-nullable.sh [hunk 8]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```
Comment (Ekberg) [inline scripts/patch-x-nullable.sh:1]: Unused script, should be removed.

**Review by Ekberg — CHANGES_REQUESTED** [review]

**Review by Ekberg — APPROVED** [review]

**Pull request merged**